### ISSUE TITLE
Login "stay signed in"

### ISSUE THREAD

**Issue by RaymondSchnyder** [thelounge/lounge#1456]
Hello,

I would like to ask if its possible to deactivate this feature / to modify it.
![image](https://user-images.githubusercontent.com/12892508/29709650-719b9332-898d-11e7-804a-05e614ae7e3c.png)

That the "Stay signed in" Checkbox is always checked, is from my perspective, a bad idea.
It would be better, to leave it unchecked for a better user expierence.

Feel free to correct me on this.

**Comment by xPaw** [comment]
If you really want to, you can edit this line:
https://github.com/thelounge/lounge/blob/d4e02d96fa4a9d6d25262b552e9ddabae50e9952/client/index.html#L102

**Comment by astorije** [comment]
Hey @RaymondSchnyder, thanks for your interest 😊

However, when it comes to user experience actually, I disagree: by default, most users will not want to be signed out when closing their browser.
I checked with a bunch of website and most often, that "Remember me option" is checked in. In fact, most websites I have checked do not even have such an option (!) and none of them ask me to re-login when I close and re-open my browser, meaning they assume that users want to stay logged in by default.

Closing as it is unlikely we will go in that direction (I would actually be okay removing it like these other websites) and @xPaw gave you an alternative if you decide you want to apply this locally :)

Thanks!
